src/commands/executeFunction/EventGridCodeLensProvider.ts

```diff
@@ -0,0 +1,14 @@
+import { CodeLens, CodeLensProvider, Range } from 'vscode';
```

Comment:
Every new file that you created needs the license at the top. You can refer to any of the files in the Functions repo for it.